ptx/sec_deriv_interpret.ptx

```diff
@@ -564,11 +569,12 @@
           <caption>Zooming in on <m>f</m> and its tangent line at <m>x=3</m> for the function given in <xref ref="ex_der_meaning4">Examples</xref> and <xref ref="ex_der_meaning5"/></caption>
           <!-- START figures/fig_fwithderivzoom3.tex -->
           <image xml:id="img_fwithderivzoom3" width="47%">
-          <shortdescription>starting at y=3 the curve arches upwards making a projectile. the tangent line of the curve is drawn at (3,4) </shortdescription>
+          <shortdescription> Zooming in on f and its tangent line at x=3 for the function given in example 2.2.7 and 2.2.10 </shortdescription>
```

Comment:
Don't hard-code numbers like Example 2.2.7 in the short description, since those numbers could change. Can you find another way to reference them?

Reply:
@cr1nja you still need to address this.